pyega3/pyega3.py

```diff
@@ -7,15 +7,15 @@
 import platform
 import random
 
-from libs.auth_client import AuthClient
-from libs.credentials import Credentials
-from libs.data_client import DataClient
-from libs.server_config import ServerConfig
-from libs.utils import get_client_ip
-from libs.data_file import DataFile
-from libs.commands import execute_subcommand
-
-version = "3.4.1"
+from pyega3.libs.auth_client import AuthClient
+from pyega3.libs.credentials import Credentials
+from pyega3.libs.data_client import DataClient
+from pyega3.libs.server_config import ServerConfig
+from pyega3.libs.utils import get_client_ip
+from pyega3.libs.data_file import DataFile
+from pyega3.libs.commands import execute_subcommand
+
+version = "3.4.2"
```

Comment:
This should either be 3.4.2 or 3.5.0 (if the renaming of the `--saveto` command-line argument is deemed to be a breaking change).